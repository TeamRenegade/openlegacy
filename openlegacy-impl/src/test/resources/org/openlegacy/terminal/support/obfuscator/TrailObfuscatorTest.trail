<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<terminal-trail>
	<snapshot type="INCOMING" sequence="1">
	    <row rowNumber="1">
	        <field value="title no mix" column="15" />
	    </row>
	    <row rowNumber="2">
	        <field value="number:" column="15" />
	        <field value="1,234.00" column="25" />
	    </row>
	    <row rowNumber="3">
	        <field value="input mix:" column="15" />
	        <field value="john doe" editable="true" column="25" />
	    </row>
	    <row rowNumber="4">
	        <field value="empty input no mix:" column="15" />
	        <field value="  " column="40" editable="true" />
	    </row>
	    <size rows="24" columns="80"/>
	</snapshot>
	<snapshot type="OUTGOING" sequence="2" command="[enter]">
	    <row rowNumber="1">
	        <field value="title no mix" column="15" />
	    </row>
	    <row rowNumber="2">
	        <field value="number:" column="15" />
	        <field value="1,234.00" column="25" />
	    </row>
	    <row rowNumber="3">
	        <field value="input mix:" column="15" />
	        <field value="jon doey" editable="true" column="25" modified="true"/>
	    </row>
	    <row rowNumber="4">
	        <field value="empty input no mix:" column="15" />
	        <field value="  " column="40" editable="true" />
	    </row>
	    <size rows="24" columns="80"/>
	</snapshot>
</terminal-trail>