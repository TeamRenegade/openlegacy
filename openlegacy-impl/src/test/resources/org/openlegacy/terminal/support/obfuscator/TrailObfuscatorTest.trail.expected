<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<terminal-trail>
    <snapshot sequence="1" type="INCOMING">
        <row rowNumber="1">
            <field column="15" value="title no mix"/>
        </row>
        <row rowNumber="2">
            <field column="15" value="number:"/>
            <field column="25" value="8,330.03" modified="true"/>
        </row>
        <row rowNumber="3">
            <field column="15" value="input mix:"/>
            <field column="25" value="kxty qhj" modified="true" editable="true"/>
        </row>
        <row rowNumber="4">
            <field column="15" value="empty input no mix:"/>
            <field column="40" value="  " modified="true" editable="true"/>
        </row>
        <size rows="24" columns="80"/>
    </snapshot>
    <snapshot sequence="2" type="OUTGOING" command="[enter]">
        <row rowNumber="1">
            <field column="15" value="title no mix" modified="true"/>
        </row>
        <row rowNumber="2">
            <field column="15" value="number:" modified="true"/>
            <field column="25" value="8,330.03" modified="true"/>
        </row>
        <row rowNumber="3">
            <field column="15" value="input mix:" modified="true"/>
            <field column="25" value="jon doey" modified="true" editable="true"/>
        </row>
        <row rowNumber="4">
            <field column="15" value="empty input no mix:" modified="true"/>
            <field column="40" value="  " modified="true" editable="true"/>
        </row>
        <size rows="24" columns="80"/>
    </snapshot>
</terminal-trail>
